thermal/Thermal.cpp

```diff
@@ -55,7 +56,11 @@ static const char *CPU_LABEL[] = {"CPU0",
                                   "CPU8",
                                   "CPU9",
                                   "CPU10",
-                                  "CPU11"};
+                                  "CPU11",
```

Comment:
currently, are we using more than 12 CPU core for any of the platform?

Reply:
Hi This is for the future. 
As I got informed by Fonn Swee Yee, that intel will support 16 cores. 
I re-verified with JaiKrishna nellamapudi. So added support till 16 CPU cores.

Reply:
No problem extending this to 16.
But it's a good idea to change this when you really have such platform where you can verify this.